monolith/views/home.py

```diff
@@ -19,10 +19,15 @@ def _strava_auth_url(config):
 
 @home.route('/')
 def index():
+    avgSpeed = 0
     if current_user is not None and hasattr(current_user, 'id'):
         runs = db.session.query(Run).filter(Run.runner_id == current_user.id)
+        for run in runs:
+            avgSpeed = avgSpeed + run.average_speed
+        avgSpeed = avgSpeed / runs.count()
     else:
         runs = None
```

Comment:
We have to handle the case in which the run count is zero. The actual code will rise a division by zero exception